base/deprecated.jl

```diff
@@ -80,7 +80,7 @@ function firstcaller(bt::Array{Ptr{Void},1}, funcsym::Symbol)
         if lkup === ()
             continue
         end
-        fname, file, line, fromC = lkup
+        fname, file, line, inlined_file, inlined_line, fromC = lkup
```

Comment:
What do the `inlined_file` and `inlined_line` fields mean? It seems potentially annoying to deal with extra information about inlining for each source location. I would expect inlined frames to be represented the same as normal ones, perhaps with an `inline` flag set to `true` or something.